app/playground/src/PlaygroundList.js

```diff
@@ -0,0 +1,186 @@
+ * So name = <ComponentName> is fine in most cases.
+ * DatePicker has as Component Wrapping it called PickerWrapper => render(<PickerWrapper>{...}</PickerWrapper>)
+ * so it needs one name to identify the component and one name to navigate correctly
+ */
+const playgroundComponents: PlaygroundComponent[] = [
+  {
+    name: 'ButtonPopup',
+    component: ButtonPopup,
+  },
+  {
+    name: 'Slider',
+    component: Slider,
+  },
+  {
+    name: 'SimpleCard',
+    component: SimpleCard,
+    navigationName: 'SimpleCardWrapper',
+  },
+  {
+    name: 'IncrementDecrementButtons',
+    component: IncrementDecrementButtons,
+  },
+  {
+    name: 'Button',
+    component: Button,
+  },
+  {
+    name: 'AdaptableBadge',
+    component: AdaptableBadge,
+  },
+  {
+    name: 'Text',
+    component: Text,
+  },
+  {
+    name: 'ReadMore',
+    component: ReadMore,
+  },
+  {
+    name: 'RoomPicker',
+    component: RoomPicker,
+  },
+  {
+    name: 'RoomDescription',
+    component: RoomDescription,
+  },
+  {
+    name: 'RoomBadges',
+    component: RoomBadges,
+  },
+  {
+    name: 'BookNow',
+    component: BookNow,
+  },
+  {
+    name: 'StarsPopup',
+    component: StarsPopup,
+  },
+  {
+    name: 'PhotoStripeHeader',
+    component: PhotoStripeHeader,
+  },
+  {
+    name: 'StarsFilter',
+    component: StarsFilter,
+  },
+  {
+    name: 'SuggestionList',
+    component: SuggestionList,
+  },
+  {
+    name: 'RecentSearches',
+    component: RecentSearches,
+  },
+  {
+    name: 'LoadMoreButton',
+    component: LoadMoreButton,
+  },
+  {
+    name: 'DatePicker',
+    component: DatePicker,
+    navigationName: 'PickerWrapper',
+  },
+].sort((a, b) => {
+  if (a.name < b.name) {
+    return -1;
+  }
+  if (a.name > b.name) {
+    return 1;
+  }
+  return 0;
+});
+
+export class PlaygroundList extends React.Component<Props> {
+  static navigationOptions = () => ({
+    title: 'Playground components',
+  });
+
+  componentDidUpdate = () => {
+    if (this.props.storageValue) {
+      this.navigateToPlayground(this.props.storageValue);
+    }
+  };
+
+  keyExtractor = (item: PlaygroundComponent) => item.name;
+
+  renderItem = ({ item }: { item: PlaygroundComponent }) => (
+    <PlaygroundListItem
+      navigationName={item.navigationName}
+      name={item.name}
+      onPress={this.navigateToPlayground}
+    />
+  );
+
+  navigateToPlayground = (name: string) => {
+    this.props.saveToStorage(name);
+    this.props.navigation.navigate({
+      routeName: 'Playground',
+      key: 'key-Playground',
+      params: {
+        name,
+        onGoBack: this.onBackClicked,
+      },
+    });
+  };
+
+  onBackClicked = () => {
+    this.props.saveToStorage(null);
+  };
+
+  render = () => (
+    <View>
+      <FlatList
+        data={playgroundComponents}
+        keyExtractor={this.keyExtractor}
+        renderItem={this.renderItem}
+        bounces={false}
+      />
```

Comment:
Without the wrapping `View` and `bounces={false}` I could not click on list items after I scrolled to bottom. 🤷‍♂️